pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -101,27 +101,34 @@ type ReconcileMigStorage struct {
 func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	var err error
 	log.Reset()
+	log.SetValues("storage", request)
 
 	// Fetch the MigStorage instance
 	storage := &migapi.MigStorage{}
 	err = r.Get(context.TODO(), request.NamespacedName, storage)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			log.V(-1).Info("Unable to get migstorage, not found.")
 			return reconcile.Result{}, nil
 		}
-		log.Trace(err)
+		log.Error(err, "Unable to get migstorage.")
 		return reconcile.Result{Requeue: true}, nil
 	}
 
 	// Report reconcile error.
 	defer func() {
-		if err == nil || errors.IsConflict(err) {
+		storage.Status.Conditions.Log(log)
+		switch {
+		case err == nil:
+			return
+		case errors.IsConflict(err):
+			log.Error(err, "Unable to update migstorage.")
```

Comment:
`Conflict` errors are routine and occur often.  We made a deliberate decision not to log them.  Especially with a stack trace.

Reply:
make sense. will drop this